test/library/packages/URL/regionArgReader.good

```diff
@@ -1,4 +1,4 @@
-<!doctype
-doctype html>
+thi
+this
 
-doctype
+thi
```

Comment:
Weird, my second comment didn't show up. I was going to suggest putting the contents of the website into `test.txt` to avoid changing the .good file (not a strong preference though).